extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -63,16 +66,40 @@ export const RTC_ERROR_ENUM = {
 /**
  * @param {string} error
  * @param {string} callout
+ * @param {!Window} win
  * @param {number=} opt_rtcTime
  * @return {!Promise<!rtcResponseDef>}
  * @private
  */
-function buildErrorResponse_(error, callout, opt_rtcTime) {
+function buildErrorResponse_(
+  error, callout, errorReportingUrl, win, ampDoc, opt_rtcTime) {
   dev().warn(TAG, `RTC callout to ${callout} caused ${error}`);
+  if (errorReportingUrl) {
+    sendErrorMessage(error, errorReportingUrl, win, ampDoc);
+  }
   return Promise.resolve(/**@type {rtcResponseDef} */(
     {error, callout, rtcTime: opt_rtcTime || 0}));
 }
 
+/**
+ * @param {string} errorType Uses the RTC_ERROR_ENUM above.
+ * @param {string} errorReportingUrl
+ * @param {!Window} win
+ */
+function sendErrorMessage(errorType, errorReportingUrl, win, ampDoc) {
+  if (Math.random() < RTC_ERROR_REPORTING_FREQUENCY) {
```

Comment:
Can you make this determined per page instead of per error report?  Put the following as a static field:

const ERROR_REPORTING_ENABLED = Math.random() < 0.01;